packages/discovery/src/local-peer-cache/index.spec.ts

```diff
@@ -11,14 +11,21 @@ import { LocalStoragePeerInfo } from "@waku/interfaces";
 import chai, { expect } from "chai";
 import chaiAsPromised from "chai-as-promised";
 import { MemoryDatastore } from "datastore-core/memory";
-import { LocalStorage } from "node-localstorage";
 import sinon from "sinon";
 
 import { LocalPeerCacheDiscovery } from "./index.js";
 
 chai.use(chaiAsPromised);
 
-global.localStorage = new LocalStorage("./mock_local_storage");
+// dynamically importing the local storage polyfill for node
+if (typeof window === "undefined") {
+  try {
+    const { LocalStorage } = await import("node-localstorage");
```

Comment:
it's only needed for `tests` so another check can be used too, for example https://github.com/waku-org/js-waku/blob/fb41f4c723238628f398dff4bc423edee8fd1c6e/packages/sdk/src/utils/libp2p.ts#L40

Reply:
`typeof window` check feels more robust for karma compilation purposes: karma can't compile node packges, thus this acts as a good safe guard.
Is this a nit? can also try with the `NODE_ENV` flag